draft-ietf-quic-recovery.md

```diff
@@ -598,9 +598,10 @@ min_rtt:
 : The minimum RTT seen in the connection, ignoring ack delay.
 
 max_ack_delay:
-: The maximum ack delay in an incoming ACK frame for this connection.
-  Excludes ack delays for non-retransmittable packets and those
-  that create an RTT sample less than min_rtt.
+: The value from the `max_ack_delay` transport parameter, which
+  indicates the maximum amount of time in milliseconds the receiver
```

Comment:
the maximum amount of time by which the receiver intends to delay acknowledgments, in milliseconds